ui/src/app/streams/streams.service.ts

```diff
@@ -271,4 +276,25 @@ export class StreamsService {
       .catch(this.errorHandler.handleError);
   }
 
+  /**
+   * Stream History
+   * @param {string} streamDefinition
+   * @returns {Observable<StreamHistory[]>}
+   */
+  getHistory(streamDefinition: string): Observable<StreamHistory[]> {
```

Comment:
Can we change the `streamDefinition` to `streamName` as it is just the `name` of the stream as the parameter?